python/tank/authentication/sso_saml2/utils.py

```diff
@@ -11,6 +11,9 @@
 SSO/SAML2 Utility functions.
 """
 
+# pylint: disable=invalid-name
+# pylint: disable=unused-import
```

Comment:
Instead of disabling a warning that might catch bad code, we add a `#noqa` at the end to the offending line instead.

Reply:
The noqa primitive works for my flake8, but not my pylint. Does it work for both in your case ?

Reply:
I've modified my use of the invalid-name disabling to just before the offending (but correct) lines.